.github/pre-commit

```diff
@@ -38,7 +38,7 @@ else
     done
 fi
 
-golangci-lint run --build-tags integration,containers_image_storage_stub
+golangci-lint run --build-tags integration,containers_image_storage_stub,e2e
```

Comment:
instead of duplicating the commands, we should use `make go/lint` and `make go/test` respectively

Reply:
this file is pretty out of date 🙂

Reply:
we will do the rework in a separate PR but get this out of the door now as it causes some considerable friction in day-to-day dev work